econet_base.yaml

```diff
@@ -74,7 +75,7 @@ econet:
     1: ${econet_alarm_update_interval}
     2: ${econet_alarm_history_update_interval}
     3: ${econet_alarm_history_update_interval}
-  src_address: 0x340
+  src_address: ${wifi_module_address}
```

Comment:
I haven't updated my water heater for likely over a month. I think this has already broken water heaters. @barndawgie I think your keep having some Wi-Fi disconnects in the alarm history. Can you test this?

Reply:
I'm running the latest build. I actually had an alert last night and everything worked as expected.

Reply:
Thanks for checking. Since we are requesting these ourselves from 0x340 this shouldn't be broken for anyone. @stockmopar can you update https://github.com/esphome-econet/esphome-econet/blob/a5cd4496ff70da41ad6f8e1be801c4c575092e3e/econet_base.yaml#L77C16-L77C21 to use the new substitution?

Reply:
Never mind. I thought this line was adding src_address to the alarm_history_1 etc. sensors. I guess I need to get some sleep...